src/main/java/edu/uml/diet/RegistrationServlet.java

```diff
@@ -0,0 +1,57 @@
+package edu.uml.diet;
+
+import javax.servlet.*;
+import javax.servlet.http.*;
+import java.io.*;
+
+/**
+ * Created by adil on 3/1/15.
+ */
+public class RegistrationServlet extends HttpServlet {
+
+    private String email = "";
+    private String password = "";
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        response.setContentType("text/html");
+        PrintWriter out = response.getWriter();
+        out.println("<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.01 Transitional//EN\"\n" +
+                "        \"http://www.w3.org/TR/html4/loose.dtd\">");
+        email = request.getParameter("email");
+        password = request.getParameter("password");
+
+        BasicUserService basicUserService = new BasicUserService(new DbUserServices());
+        //TODO: remove
+        DatabaseBuilder builder = new DatabaseBuilder(new DatabaseConnector(), "DietTracker");
+        try {
+            if(!builder.CheckIfDbExists()) {
+                builder.CreateDatabase();
+            }
+        } catch (DatabaseConnectorException e) {
+            e.printStackTrace();
+        }
+        try {
+            if(!builder.CheckIfTableExists("USERS")) {
+                builder.CreateUserTable();
+            }
+        } catch (DatabaseConnectorException e) {
+            e.printStackTrace();
```

Comment:
clean up exception handling here too. (anywhere there is a catch that doesn't actually handle the exception).